src/script/auth/component/PhoneLoginForm.tsx

```diff
@@ -77,7 +81,12 @@ const PhoneLoginForm = ({isFetching, onSubmit}: LoginFormProps) => {
     <div>
       <Select
         id="select-phone"
-        onChange={onSelectCountry}
+        onChange={country => {
+          const selectedCountry = country as Option;
+          const selectedCountryValue = selectedCountry.value as string;
+          setCountry(selectedCountryValue);
+          setCountryCode((getCountryCode(selectedCountryValue) || 'X2').toString(10));
+        }}
```

Comment:
this doesnt work outside of the Select component? its a lot of lines for an inline function

Reply:
to be honest i dont like this. if our select component is forcing us to do stupid shit we should change the component not hack the consumer to fit it.

Reply:
https://github.com/wireapp/wire-web-packages/pull/4351